Fix dependency on redis-cli and redis-server to pass tests (#3014)
valkey-io/valkey-glide: node/DEVELOPER.md

source "$HOME/.cargo/env"
```

**Valkey Server and CLI**

> For testing purposes, you need to install the Valkey server and CLI. You can follow the instructions [here](https://valkey.io/topics/installation/).

Reviewer: why not make it similar to go and c#?
```suggestion
See the [Valkey installation guide](https://valkey.io/topics/installation/) to install the Valkey server and CLI.
```
